cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -560,11 +560,19 @@ def _measurement(two_qubit_circuit: circuits.Circuit) -> np.ndarray:
 
 
 def _create_parallel_rb_circuit(
-    qubits: Iterator['cirq.Qid'], num_cliffords: int, c1: list
+    qubits: Sequence['cirq.Qid'], num_cliffords: int, c1: list
 ) -> 'cirq.Circuit':
-    circuits_to_zip = [_random_single_q_clifford(qubit, num_cliffords, c1) for qubit in qubits]
-    circuit = circuits.Circuit.zip(*circuits_to_zip)
-    return circuits.Circuit.from_moments(*circuit, ops.measure_each(*qubits))
+    sequences_to_zip = [_random_single_q_clifford(qubit, num_cliffords, c1) for qubit in qubits]
+    # Ensure each sequence has the same number of moments.
+    num_moments = max(len(sequence) for sequence in sequences_to_zip)
+    for i in range(len(sequences_to_zip)):
+        if len(sequences_to_zip[i]) < num_moments:
+            sequences_to_zip[i].extend(
+                [ops.SingleQubitCliffordGate.I(qubits[i])]
+                * (num_moments - len(sequences_to_zip[i]))
+            )
+    moments = tuple(zip(*sequences_to_zip, strict=True))
```

Comment:
nit: don't need to convert to tuple here

Reply:
done